interview-questions.md

```diff
@@ -6,38 +6,50 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
+1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model? 
 
-Your answer:
+Your answer:go back and add the foreign edit it so it has the foreign key. The foreign key is the id of the colums in the second table. it would be on the student model
 
-Researched answer:
+Researched answer: a foreign key is a column of the table which belongs to the other one. Using migrate can help change or add a column if the developer forgot to add one. The name of the foreign key is the name of the column, it would be the student model since it belongs to the cohort table.
 
 2. Which RESTful routes must always be passed params? Why?
 
-Your answer:
+Your answer: the ones that invlolve seeing, creating, updating and deleting them so the computer and/or software knows what to see, change, create, update and delete. this would be view, put, post , delete, and patch.
 
-Researched answer:
+Researched answer: I couldnt find any online sources that informed me of the required routes that needed params
```

Comment:
The RESTful routes that must be passed params, specifically in the URL are show, update, edit, and destroy.  You can find this in the syllabus in the [RESTful Routes](https://github.com/learn-academy-2022-foxtrot/Syllabus/blob/main/rails/restful-routes-crud.md) section.